ink/src/main/java/com/simplify/ink/InkView.java

```diff
@@ -96,6 +96,8 @@
     RectF dirty;
     ArrayList<InkListener> listeners = new ArrayList<>();
 
+    private boolean mIsEmpty;
```

Comment:
We've trended away from using mVariable naming convention. Can you rename this 'isEmpty' ?